templates/openstack/consulting.html

```diff
@@ -1,389 +1,501 @@
-              attrs={"class": "p-inline-images__logo"},
-              loading="lazy",
+        </div>
+        <div class="p-logo-section__item">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/5af38159-etisalat_logo.svg",
+            alt="",
+            width="108",
+            height="41",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "logo-section__logo"}
             ) | safe
           }}
-        </li>
-        <li class="p-inline-images__item">
-          {{
-            image(
-              url="https://assets.ubuntu.com/v1/4d6054f9-logo-cisco.svg",
-              alt="Cisco",
-              height="76",
-              width="144",
-              hi_def=True,
-              attrs={"class": "p-inline-images__logo"},
-              loading="lazy",
+        </div>
+        <div class="p-logo-section__item">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/f7651dc9-Logo-GARR-300dpi.png",
+            alt="",
+            width="2417",
+            height="917",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "logo-section__logo"}
             ) | safe
           }}
-        </li>
-        <li class="p-inline-images__item">
-          {{
-            image(
-              url="https://assets.ubuntu.com/v1/b7693339-logo-bloomberg.svg",
-              alt="Bloomberg",
-              height="28",
-              width="144",
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-6">
+      <h2>OpenStack implementation challenges</h2>
+      <p>Designing an optimal architecture for OpenStack implementation that maximises price-performance requires expert knowledge and years of experience.</p>
+      <p>Here are some common questions one has to ask when implementing OpenStack:</p>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">How to configure the hardware to meet performance and capacity requirements, while not exceeding the budget?</li>
+        <li class="p-list__item is-ticked">Which OpenStack architecture to choose to maximise resource consumption and cost-efficiency of the cloud?</li>
+        <li class="p-list__item is-ticked">Which hypervisor, software-defined networking (SDN) and storage platform to choose?</li>
+        <li class="p-list__item is-ticked">How to design the cloud to meet local security and compliance regulations?</li>
+        <li class="p-list__item is-ticked">How to migrate the existing workloads from VMware or other cloud environments?</li>
+      </ul>
+      <p>Canonical's private cloud consulting services solve all those challenges, providing a comprehensive implementation package for Charmed OpenStack at a fixed price.</p>
+      <p><a href="/openstack/features">Learn more about Charmed OpenStack&nbsp;&rsaquo;</a></p>
+    </div>
+    <div class="col-6 u-align--center u-vertically-center">
```

Comment:
```suggestion
    <div class="col-6 u-align--center u-vertically-center u-hide--small">
```
Very large on small screens